packages/indy-vdr/src/IndyVdrModuleConfig.ts

```diff
@@ -1,6 +1,45 @@
 import type { IndyVdrPoolConfig } from './pool'
+import type { IndyVdr } from '@hyperledger/indy-vdr-shared'
 
 export interface IndyVdrModuleConfigOptions {
+
+    /**
+   *
+   * ## Node.JS
+   *
+   * ```ts
+   * import * as indyVdr from '@hyperledger/indy-vdr-nodejs';
+   * 
+   * const agent = new Agent({
+   *  config: {},
+   *  dependencies: agentDependencies,
+   *  modules: {
+   *   indyVdr: new IndyVdrModule({
+   *      indyVdr,
+   *   }) 
+   *  }
+   * })     
+   * ```
+   *
+   * ## React Native
+   *
+   * ```ts
+   * import * as indyVdr from '@hyperledger/indy-vdr-react-native';
+   * 
+   * const agent = new Agent({
+   *  config: {},
+   *  dependencies: agentDependencies,
+   *  modules: {
+   *   indyVdr: new IndyVdrModule({
+   *      indyVdr,
+   *   }) 
+   *  }
+   * })     
+   * ```
+   * ```
+   */
+    indyVdr: IndyVdr
```

Comment:
I think you need to update all places that create an instance of the module (`new IndyVdrModule`) to provide this parameter

Reply:
@TimoGlastra the indyVdr arguement will come from the @hyperledger/indy-vdr-shared or @hyperledger/indy-vdr-nodejs?